lib/src/cache_manager.dart

```diff
@@ -26,7 +27,7 @@ class FirebaseImageCacheManager {
 
   Future<void> open() async {
     db = await openDatabase(
-      join((await getDatabasesPath())!, dbName),
+      join((await getDatabasesPath()), dbName),
```

Comment:
I get the error `The argument type 'String?' can't be assigned to the parameter type 'String'.dart(argument_type_not_assignable)` here... Any ideas?

Reply:
That's odd, I don't think it returns a nullable [string anymore](https://github.com/tekartik/sqflite/blob/81c0c30aa59d7d3f229bf6ce0494458f96f893c2/sqflite_common/test/src_mixin_test.dart#L114).

Looking at ^2.0.0+3 of sqflite, all their uses of `getDatabasesPath` have a return type of `Future<String>`